### ISSUE TITLE
More gracefully handle error on no space left

### ISSUE THREAD

**Issue by haatveit** [unioslo/tsd-file-api#239]
Currently we fail out with generic 500 when an OSError is raised for the target partition being out of space. It would be nice returning specific errors for being out of space or quota.

**Comment by leondutoit** [comment]
Yeah, we can add `errno.ENOSPC` to https://github.com/unioslo/tsd-file-api/blob/e8bf6102522d15da84883dd94521c511334d591f/tsdfileapi/exc.py#L198
